test/Api.Test/Api.Test.csproj

```diff
@@ -26,4 +26,12 @@
     <ProjectReference Include="..\Core.Test\Core.Test.csproj" />
   </ItemGroup>
 
+  <ItemGroup>
+    <None Remove="Auth\" />
+    <None Remove="Auth\Controllers\" />
+  </ItemGroup>
```

Comment:
❓ This seems generated and could maybe be removed?

Reply:
Removed the first `ItemGroup` and its `None` entries